doc/coq.rst

```diff
@@ -27,9 +27,7 @@ version should be selected in the :ref:`dune-project` file. For example:
     (using coq 0.3)
 
 This will allow for the ``coq.theory`` stanza to be included in the current
-project. If the language version is absent, Dune will automatically add this
-line with the latest Coq version to the project file once a ``(coq.theory ...)``
-stanza is used anywhere.
```

Comment:
@ejgallego I tested this and it doesn't seem to be the case so removing.

Reply:
This misfeature was removed in 3.0